data/fields/bollard.json

```diff
@@ -7,7 +7,7 @@
             "fixed": "Fixed Bollard",
             "flexible": "Flexible Bollard",
             "foldable": "Foldable Bollard",
-            "removable": "Removable Bollard",
+            "removable": "Removable Bollard (with or without a key)",
```

Comment:
```suggestion
            "removable": { "title": "Removable Bollard", "description: Removable with or without a key" },
```

Reply:
or should this PR be closed as an abandoned?